supervisor/api/__init__.py

```diff
@@ -198,6 +198,7 @@ def _register_host(self) -> None:
                 web.post("/host/reload", api_host.reload),
                 web.post("/host/options", api_host.options),
                 web.get("/host/services", api_host.services),
+                web.get("/host/disk/default/usage", api_host.disk_usage),
```

Comment:
For other API we use the plural, so `disks`.

We also discussed if we should fold the usage and other metrics (e.g. lifetime) into a single endpoint. Usually the frontend prefers if not too many calls need to be made. So this could just become `/host/disk/default`, but then also return the lifetime, and maybe even partition layout.

But then again, looking at the current implementation, we already have a query parameter. And I guess since getting exact usage can become an expensive operation, maybe it is better to have this as a separate endpoint.

@MindFreeze if you/the frontend team is fine to have it as a separate endpoint, I am fine too.